tests/manager/conftest.py

```diff
@@ -470,6 +464,7 @@ async def clean_fixture():
                 await conn.execute((users.delete()))
                 await conn.execute((scaling_groups.delete()))
                 await conn.execute((domains.delete()))
+                await conn.execute(((await load_table(engine, "container_registries")).delete()))
```

Comment:
Why does this line use a specially crafted `load_table()` function?

Reply:
Since the `container_registries` variable was not directly defined as a table, I wrote it this way to import the ContainerRegistry table object.

It would be nice if we could remove `load_table` if possible.